src/main/java/com/exadel/discountwebapp/location/repository/LocationRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.exadel.discountwebapp.location.repository;
+
+import com.exadel.discountwebapp.location.entity.Location;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface LocationRepository extends CrudRepository<Location, Long> {
+
+    @Override
+    Optional<Location> findById(Long id);
```

Comment:
it seems like this method is provided by the CrudRepository, no need to define it here

Reply:
removed